tests/marine/test_marine.py

```diff
@@ -272,13 +286,38 @@ def test_http_packet_filter_and_parse(marine_or_marine_pool: Union[Marine, Marin
     general_filter_and_parse_test(
         marine_or_marine_pool=marine_or_marine_pool,
         packet=packet.bin(),
+        packet_encapsulation=ENCAP_TYPE_ETHERNET,
         bpf_filter=bpf_filter,
         display_filter=display_filter,
         expected_passed=True,
         expected_output=expected_output,
     )
 
 
+def test_radiotap_packet_filter_and_parse(
+    marine_or_marine_pool: Union[Marine, MarinePool]
+):
+    packet_data = b"\x00\x00\x12\x00\x2e\x48\x00\x00\x00\x02\x6c\x09\xa0\x00\xc2\x07\x00\x00\xff\xff"
+    display_filter = "radiotap"
+    expected_output = {
+        "radiotap.present.tsft": 0,
+        "radiotap.present.channel": 1,
+        "radiotap.present.rate": 1,
```

Comment:
Please include some l3 fields to verify that it is parsed correctly (`ip,src` will suffice)